src/features/profile/Profile.tsx

```diff
@@ -22,40 +27,75 @@ interface ProfileProps {
   navigation: NavigationProp;
 }
 
-const profile = ({navigation}: ProfileProps) => {
-  const logOutHandler = () =>
+interface Props extends ProfileProps {
+  logOutHandler: () => void;
+  changeTheme: (isDark: boolean) => void;
+  isDark: boolean;
+}
+
+const profile: React.FC<Props> = ({
+  navigation,
+  logOutHandler,
+  changeTheme,
+  isDark,
+}) => {
+  const toggleSwitch = (isDark: boolean) => {
+    changeTheme(isDark);
```

Comment:
you can use here theme enum not bool value to determine more themes in future ...